lib/bowling.rb

```diff
@@ -0,0 +1,49 @@
+class Bowling
+  
+  def initialize
+    @rolls = []
+  end
+
+  def rollball(pins)
+    @rolls << pins
+  end
+
+  def score
+    result = 0
+    number_of_roll = 0
+    10.times do
```

Comment:
This assumes you must have 1 complete games, perhaps as a future dev could incorporate removing the score at any stage after a completed frame?